beetsplug/embedart.py

```diff
@@ -117,15 +123,10 @@ def embed_item(self, item, imagepath, maxwidth=None, itempath=None,
         if compare_threshold:
             if not self.check_art_similarity(item, imagepath,
                                              compare_threshold):
-                self._log.warn(u'Image not similar; skipping.')
+                self._log.debug(u'Image not similar; skipping.')
```

Comment:
I'd lean `info` on this one over `debug`: you probably would want to know this when running `beet fetchart` directly.